src/index.js

```diff
@@ -0,0 +1,132 @@
+const state = {
+  temp: 68
+};
+
+let isFahrenheit = true;
+
+const vibe = {
+  sheet: 2
+}
+
+let isReset = false;
+
+const changeTemp = (direction) => {
+  if (direction === 'up') {
+    state.temp += 1;
+  } else if (direction === 'down') {
+    state.temp -= 1;
+  } else if (direction === null) {
+    pass
+  }
+
+  const temperature = document.querySelector("#temperature");
+  if (isFahrenheit) {
+    temperature.textContent = `${state.temp}℉`;
+  } else {
+    temperature.textContent = `${(state.temp -32) * 5/9}℃`;
+  }
+  changeTempFontColor();
+}
+
+
+const swapStyleSheet = (event) => {
+
+  if (vibe.sheet === 1 || vibe.sheet === 2) {
+      vibe.sheet += 1;
+  } else if (vibe.sheet === 3) {
+      vibe.sheet = 1;
+  }
```

Comment:
To cycle through a set of integer values (counting up and wrapping around) modulo arithmetic can help. Consider the following:

```js
  vibe.sheet = (vibe.sheet + 1) % numSheets;
```

This will count up, and when the sheet count equals `numSheets` it will cycle back around to 0. (Either number the sheets starting from 0, or add 1 when building the sheet name.)